.github/dependabot.yml

```diff
@@ -42,3 +39,16 @@ updates:
   - dependency-name: MSTest.TestAdapter
     versions:
     - 2.2.2
+
+- package-ecosystem: nuget
+  directory: "/src/CurrentFileVersionCompatibilityTests"
+  schedule:
+    interval: daily
+    time: "13:00"
```

Comment:
I'm curious, do we need this in here given we also have this in the top level scope?

Reply:
I honestly don't know. I couldn't find any dependabot.yml files that try to ignore within a subtree, so this whole thing is sort of experimental. Let's try it this way for now and see if it works at all.